### PR TITLE
Dashboard graphs paula

### PR DESCRIPTION
**Pull request by paulauus**
The dashboard now has an OS selection, a calendar date selection and an NSFW tickbox.

### CODE REVIEW

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
+    # Set a specific minimum date (e.g., October 7, 2024)
+    min_date = datetime(2024, 10, 7).date()
+
+    # Create a date range selector with "All time" as the default range
+    # Default to the range from min_date to today
+    default_start_date = min_date
+    default_end_date = today
+
+    # Create the date input and store the output
+    date_range = st.date_input(
```
Comment (Leon-JS02) [inline dashboard/dashboard.py:37]: This is some fancy stuff right here. Using date ranges will make the Pandas stuff easier too.

File: dashboard/dashboard.py [hunk 2]
```diff
@@ -1,12 +1,53 @@
+# pylint: disable=C0103
```
Comment (Leon-JS02) [inline dashboard/dashboard.py:1]: What's this warning?

File: dashboard/dashboard.py [hunk 3]
```diff
@@ -1,12 +1,53 @@
...
 # Page configuration
 st.set_page_config(layout="wide")
 
-st.title("Games Tracker")
+cols = st.columns([2, 1, 1, 0.5])
+with cols[0]:
+    st.title("Games Tracker")
+
+# Create a dropdown selection
+with cols[1]:
```
Comment (Leon-JS02) [inline dashboard/dashboard.py:19]: Cool!

**Review by Leon-JS02 — APPROVED** [review]
Ran the script locally - dashboard looking nice so far.
Good job.

File: dashboard/dashboard.py (hunk 2 above)
Comment (paulauus) [reply dashboard/dashboard.py:1]: Python doesn't like the file names when they're not snake case, but we need them to be capitalised for the dashboard.

File: dashboard/dashboard.py [hunk 4]
```diff
@@ -1,12 +1,53 @@
...
+    min_date = datetime(2024, 10, 7).date()
+
+    # Create a date range selector with "All time" as the default range
+    # Default to the range from min_date to today
+    default_start_date = min_date
+    default_end_date = today
+
+    # Create the date input and store the output
+    date_range = st.date_input(
+        "Select date range:",
```
Comment (mehrdadhalali) [inline dashboard/dashboard.py:38]: Nice! Looking forward to see how this looks

**Review by mehrdadhalali — APPROVED** [review]

**Pull request merged**